### PR TITLE
KDE: new content pack update.

### PR DESCRIPTION
**Pull request by ScarlettGatelyMoore**
- [x] Have you followed the [guidelines for contributing](https://github.com/snapcore/snapcraft/blob/master/CONTRIBUTING.md)?
- [x] Have you signed the [CLA](http://www.ubuntu.com/legal/contributors/)?
- [ ] Have you successfully run `make lint`?
- [ ] Have you successfully run `pytest tests/unit`?

-----

### CODE REVIEW

File: tests/unit/extensions/test_kde_neon.py [hunk 1]
```diff
@@ -38,7 +38,7 @@ def kde_neon_extension_with_build_snap():
             "base": "core22",
             "parts": {
                 "part1": {
-                    "build-snaps": ["kf5-5-108-qt-5-15-10-core22-sdk/latest/stable"]
+                    "build-snaps": ["kf5-5-110-qt-5-15-11-core22-sdk/latest/stable"]
```
Comment (sergiusens) [inline tests/unit/extensions/test_kde_neon.py:41]: the point of this test fixture is to use something different that the default, so I would leave it in its original value

Comment (ScarlettGatelyMoore) [reply tests/unit/extensions/test_kde_neon.py]: Fixed.

**Comment by sergiusens** [comment]
there seems to still be a test failing here related to this change

**Comment by ScarlettGatelyMoore** [comment]
> there seems to still be a test failing here related to this change

Sorry. I can be real dumb sometimes. All should be green here shortly.

File: tests/spread/extensions/kde-neon/task.yaml [hunk 2]
```diff
@@ -48,7 +48,7 @@ execute: |
   if [[ "$SPREAD_SYSTEM" =~ ubuntu-20.04 ]]; then
     snap list kde-frameworks-5-99-qt-5-15-7-core20
   elif [[ "$SPREAD_SYSTEM" =~ ubuntu-22.04 ]]; then
-    snap list kf5-5-108-qt-5-15-10-core22
+    snap list kf5-5-111-qt-5-15-11-core22
```
Comment (lengau) [inline tests/spread/extensions/kde-neon/task.yaml:51]: Why is this 111 rather than 113? (I presume a typo?)

Comment (ScarlettGatelyMoore) [reply tests/spread/extensions/kde-neon/task.yaml:51]: Ah, because when I initially did the PR we were at 111. Fixing now..

**Review by lengau — APPROVED** [review]
Thanks for all this!

**Review by mr-cal — APPROVED** [review]

**Comment by lengau** [comment]
@sergiusens we'll probably need an override here. The CLA check failure is due to the github-assigned email, but @soumyaDghosh has signed the CLA (see: https://launchpad.net/~soumyadghosh)

**Comment by ScarlettGatelyMoore** [comment]
> @sergiusens we'll probably need an override here. The CLA check failure is due to the github-assigned email, but @soumyaDghosh has signed the CLA (see: https://launchpad.net/~soumyadghosh)

And so have I... https://launchpad.net/~scarlettmoore

**Review by sergiusens — APPROVED** [review]

**Pull request merged**